src/c75_menuCTRL.hpp

```diff
@@ -111,7 +115,98 @@ bool processControlKeys()
                     ctrlState = MANUAL_CONTROL_MODE;
                     mount.stopSlewing(ALL_DIRECTIONS);
                 }
-                else if ((key == btnDOWN) || (key == btnUP))
+                else if (key == btnDOWN)
+                {
+        #if USE_HALL_SENSOR_RA_AUTOHOME == 1
+                    ctrlState = HIGHLIGHT_RA_AUTO_HOME;
+        #else
+                    ctrlState = HIGHLIGHT_SERIAL;
+        #endif
+                }
+                //DEC-
+                else if (key == btnDOWN)
+                {
+        #if USE_HALL_SENSOR_DEC_AUTOHOME == 1
+                    ctrlState = HIGHLIGHT_DEC_AUTO_HOME;
+        #else
+                    ctrlState = HIGHLIGHT_SERIAL;
+        #endif
+                }
+                //-DEC
+                else if (key == btnUP)
+                {
+                    ctrlState = HIGHLIGHT_SERIAL;
+                }
+                else if (key == btnRIGHT)
+                {
+                    lcdMenu.setNextActive();
+                }
+            }
+            break;
+
+        #if USE_HALL_SENSOR_RA_AUTOHOME == 1
+
+        case HIGHLIGHT_RA_AUTO_HOME:
+            if (lcdButtons.keyChanged(&key))
+            {
+                waitForRelease = true;
+                if (key == btnSELECT)
+                {
+                    ctrlState = RUNNING_RA_HOMING_MODE;
+                    lcdMenu.printMenu("RA Homing...");
+                    mount.stopSlewing(ALL_DIRECTIONS);
+                    mount.findHomeByHallSensor(StepperAxis::RA_STEPS, -1, 2);  // Search 2hrs by default
+                }
+                else if (key == btnUP)
+                {
+                    ctrlState = HIGHLIGHT_MANUAL;
+                }
+                //DEC-
+                else if (key == btnDOWN)
+                {
+                    ctrlState = HIGHLIGHT_DEC_AUTO_HOME;
+                }
+                //-DEC
+                else if (key == btnDOWN)
```

Comment:
This also will never execute

Reply:
I have posted a video showing that it works...